### PR TITLE
merge docs/releases/v0.25.0.md into release branch

### PR DESCRIPTION
**Pull request by PaurushGarg**
**Description:** <Describe what has changed.>
merge docs/releases/v0.25.0.md and Version into release branch for release v0.25.0

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by vasireddy99 — APPROVED** [review]

File: docs/releases/v0.25.0.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+## [v0.25.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.25.0) (2023-01-05)
+
+[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.23.1...v0.25.0)
```
Comment (bryan-aguilar) [inline docs/releases/v0.25.0.md:5]:
```suggestion
[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.24.1...v0.25.0)
```

File: docs/releases/v0.25.0.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+**Closed issues:**
+
+- ECS EC2 sidecar deployment Cloudformation template [\#1591](https://github.com/aws-observability/aws-otel-collector/issues/1591)
```
Comment (bryan-aguilar) [inline docs/releases/v0.25.0.md:9]: I think we can get rid of this closed issue in favor of pull requests

File: docs/releases/v0.25.0.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+**Merged pull requests:**
+
+- Fixed incorrect docker-compose command [\#1730](https://github.com/aws-observability/aws-otel-collector/pull/1730) ([jkclark](https://github.com/jkclark))
+- Update GitHub Workflows to use IAM Roles w/ GitHub OIDC [\#1714](https://github.com/aws-observability/aws-otel-collector/pull/1714) ([bryan-aguilar](https://github.com/bryan-aguilar))
```
Comment (bryan-aguilar) [inline docs/releases/v0.25.0.md:14]:
```suggestion
```

File: docs/releases/v0.25.0.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.23.1...v0.25.0)
+
+**Closed issues:**
+
+- ECS EC2 sidecar deployment Cloudformation template [\#1591](https://github.com/aws-observability/aws-otel-collector/issues/1591)
+
+**Merged pull requests:**
+
+- Fixed incorrect docker-compose command [\#1730](https://github.com/aws-observability/aws-otel-collector/pull/1730) ([jkclark](https://github.com/jkclark))
```
Comment (bryan-aguilar) [inline docs/releases/v0.25.0.md:13]:
```suggestion
```

File: docs/releases/v0.25.0.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+- Update GitHub Workflows to use IAM Roles w/ GitHub OIDC [\#1714](https://github.com/aws-observability/aws-otel-collector/pull/1714) ([bryan-aguilar](https://github.com/bryan-aguilar))
+- Update dependencies [\#1710](https://github.com/aws-observability/aws-otel-collector/pull/1710) ([bryan-aguilar](https://github.com/bryan-aguilar))
+- Updates to ECS EC2 sidecar deployment CFN template [\#1592](https://github.com/aws-observability/aws-otel-collector/pull/1592) ([PaurushGarg](https://github.com/PaurushGarg))
+
```
Comment (bryan-aguilar) [inline docs/releases/v0.25.0.md:17]: I believe this is missing an entry for the updating of dependencies to `v0.68.0`?

**Review by bryan-aguilar — COMMENTED** [review]
Also update the `version.yaml` file version.

**Comment by PaurushGarg** [comment]
> Also update the `version.yaml` file version.

Thanks. Updated.

File: docs/releases/v0.25.0.md (hunk 1 above)
Comment (PaurushGarg) [reply docs/releases/v0.25.0.md]: Thanks very much. I missed that. Updated now.

File: docs/releases/v0.25.0.md (hunk 2 above)
Comment (PaurushGarg) [reply docs/releases/v0.25.0.md]: Updated.

File: docs/releases/v0.25.0.md (hunk 3 above)
Comment (PaurushGarg) [reply docs/releases/v0.25.0.md]: Updated.

File: docs/releases/v0.25.0.md (hunk 4 above)
Comment (PaurushGarg) [reply docs/releases/v0.25.0.md]: Updated.

File: docs/releases/v0.25.0.md (hunk 5 above)
Comment (PaurushGarg) [reply docs/releases/v0.25.0.md]: Updated.

**Review by vasireddy99 — APPROVED** [review]

**Pull request merged**